backend/api/utils.py

```
@@ -472,3 +477,68 @@ def notify_live_update(event: LiveUpdateEvent):
+    try:
+        response = requests.post(
+            url + "/api/render-email/", json=payload, headers=headers, timeout=5
+        )
+
+        if response.status_code == 200:
+            data = response.json()
+            html_content = data.get("html")
+            text_fallback = data.get("text")
+            msg_subject = data.get("subject")
+            success = True
+            logger.info(f"Successfully rendered email for template '{template_key}'")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)